collect_app/build.gradle

```diff
@@ -298,7 +301,6 @@ dependencies {
     }
 
     androidTestImplementation "com.squareup.okhttp3:mockwebserver:3.9.0"
```

Comment:
the `mockwebserver` dependency should have the same version as the main `okhttp` dependency. Would be good if they simply shared the version via a variable.

Reply:
Thanks, updated to the version number to use a variable